setup.py

```diff
@@ -0,0 +1,31 @@
+import setuptools
+
+setuptools.setup(
+	name='apple-health-exporter',
+	version='1.0.0',
+	author='Milind Ganjoo',
+	author_email='milind.ganjoo@gmail.com',
+	description='Python script to export Apple Health dump file to a data frame for analysis.',
+	long_description="""
+# Apple Health Data Exporter
+
+This Python 3 script takes a dump of Apple Health data (the `export.zip` file
+generated through an Apple Health data export) and saves a subset of fields and
+records to a [feather](https://github.com/wesm/feather) file. This can then be
+read by both Python and R.
+	""",
+	long_description_content_type="text/markdown",
+	url='https://github.com/mganjoo/apple-health-exporter',
+	project_urls = {
+		"Bug Tracker": "https://github.com/mganjoo/apple-health-exporter/issues"
+	},
+	license='MIT',
+	packages=['apple_health_exporter'],
+	package_dir={'.':''},
```

Comment:
I think this should be the other way round:

```suggestion
	package_dir={'':'.'}
```

since the key is the package name, in this case `""` or "empty package", and the value is a directory name.